internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -165,11 +189,16 @@ func resourceDatadogSingleSignOnConfigurationsUpdate(d *pluginsdk.ResourceData,
 
 	payload := singlesignon.DatadogSingleSignOnResource{
 		Properties: &singlesignon.DatadogSingleSignOnProperties{
-			SingleSignOnState: pointer.To(singlesignon.SingleSignOnStates(d.Get("single_sign_on_enabled").(string))),
-			EnterpriseAppId:   utils.String(d.Get("enterprise_application_id").(string)),
+			EnterpriseAppId: pointer.To(d.Get("enterprise_application_id").(string)),
 		},
 	}
 
+	if v, ok := d.GetOk("single_sign_on"); ok {
```

Comment:
We should refactor this update method to use `d.HasChanges` so people can use `ignore_changes` if they want